client/js/components/toggle-show-hide.js

```diff
@@ -0,0 +1,22 @@
+import showHide from './show-hide';
+import { trackEvent } from '../utils/track-event';
+
+export default (el) => {
+  const toggle = showHide({el});
+  const trackingJson = el.getAttribute('data-tracking-info');
```

Comment:
Don't worry about pedantry, but I don't follow – won't we need to get _some_ information from the rendered markup one way or another?